src/parser/csql_lexer.l

```diff
@@ -260,7 +260,9 @@ int yybuffer_pos;
 [dD][aA][yY]_[sS][eE][cC][oO][nN][dD]     				{ begin_token(yytext);   return DAY_SECOND; }
 [dD][aA][yY]_[mM][iI][nN][uU][tT][eE]     				{ begin_token(yytext);   return DAY_MINUTE; }
 [dD][aA][yY]_[hH][oO][uU][rR]	       					{ begin_token(yytext);   return DAY_HOUR; }
-[dD][bB][lL][iI][nN][kK]					        { begin_token(yytext); 	 return DBLINK; }
+[dD][bB][lL][iI][nN][kK]					        { begin_token(yytext);
+										csql_yylval.cptr = pt_makename(yytext);
+										return DBLINK; }
```

Comment:
Can I know why `DBLINK` should be a PT_NAME node?

Reply:
pt_makename() does not create PT_NODE, it just contains text.
It can be used as an identifier other than a reserved keyword, so it is passed the string "dblink".